webxrlayers-1.bs

```diff
@@ -298,6 +306,7 @@ interface XRQuadLayer : XRLayer {
 The <dfn attribute for="XRQuadLayer">transform</dfn> attributes sets and returns the offset and orientation relative to the
 <dfn attribute for="XRQuadLayer">space</dfn> attribute. The {{XRQuadLayer/transform}} and {{XRQuadLayer/space}} attributes
 establish the spatial relationship of the layer within the user’s physical environment.
+When setting the {{XRQuadLayer/space}}, first run the steps for [=setting the space on a layer=].
```

Comment:
_Technically_ all of these attributes should be reflecting internal slots that we define, but that's nbd (spec review might catch this eventually). I might make that change later.

Reply:
Is that done in the WebXR spec? If so, I can follow that pattern